tools/mo/openvino/tools/mo/utils/check_ie_bindings.py

```diff
@@ -70,7 +70,10 @@ def import_core_modules(silent: bool, path_to_module: str):
         print("{}: \t{}".format("Model Optimizer version", mo_version))
 
         versions_mismatch = False
-        if mo_version != ie_version:
+        # ie_version has a prefix from CI_BUILD_NUMBER but
+        # if mo_version is from git then it does not have that prefix,
+        # therefore smart comparison is needed
+        if mo_version not in ie_version:
```

Comment:
indeed these versions are the same
![image](https://user-images.githubusercontent.com/5703039/145435913-ca102f1e-a5f5-4012-a1f9-75167262123f.png)

Reply:
they are not actually the same.

Reply:
Please investigate why MO version started to differ from MO.

@andrei-kochin FYI ^^^

Reply:
@GlebKazantaev sorry for a bit confusing example. I wanted to emphasize that even if commit revision were the same there is a prefix in IE version, therefore it's better to compare this strings not by `==` but rather with `in`.
In the example above versions differ because I've build IE from a slightly different revision that a was at the moment. `IE` version was hardcoded while `MO` was deduced during python runtime with `git`.

But if a recompile `IE` then still there is this prefix.
![image](https://user-images.githubusercontent.com/5703039/145775013-9ceffbcc-1162-4922-9733-41e5c6f4e697.png)
And either we need to add the same prefix to MO or compare only revision versions without prefix. I preferred the latest option